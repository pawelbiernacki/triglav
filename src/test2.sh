#!/usr/bin/bash

echo "test2"

./triglav test2.triglav > test2.output

if diff test2.output test2.expected; then
	echo "OK";
else
	echo "error";
	exit -1;
fi
