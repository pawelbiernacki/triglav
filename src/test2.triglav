#!triglav

type boolean={false,true};
type person={Conan,Gotrek,Gwaigilion};
type place={Warszawa,Krakow,Wroclaw,Poznan,Gdansk};

type information={(X:person)_is_in_(Y:place)};

expand(2);

report();
