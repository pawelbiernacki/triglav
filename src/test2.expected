type boolean={false,true};
type person={Conan,Gotrek,Gwaigilion};
type place={Warszawa,Krakow,Wroclaw,Poznan,Gdansk};
type information={(X:person)_is_in_(Y:place)};
type instance boolean:{false,true};
type instance person:{Conan,Gotrek,Gwaigilion};
type instance place:{Warszawa,Krakow,Wroclaw,Poznan,Gdansk};
type instance information:{Conan_is_in_Warszawa,Gotrek_is_in_Warszawa,Gwaigilion_is_in_Warszawa,Conan_is_in_Krakow,Gotrek_is_in_Krakow,Gwaigilion_is_in_Krakow,Conan_is_in_Wroclaw,Gotrek_is_in_Wroclaw,Gwaigilion_is_in_Wroclaw,Conan_is_in_Poznan,Gotrek_is_in_Poznan,Gwaigilion_is_in_Poznan,Conan_is_in_Gdansk,Gotrek_is_in_Gdansk,Gwaigilion_is_in_Gdansk};
amount_of_actions=0
